controller/nodegroup.go

```diff
@@ -102,6 +102,14 @@ func buildLaunchTemplateData(group eksv1.NodeGroup, ec2Service *ec2.EC2) (*ec2.R
 		}
 	}
 
+	// For a non-default VPC, you must use security group IDs. You cannot specify both a security group ID and security
+	// name in the same request.
+	// So we grab the security group IDs to add to the launch template.
```

Comment:
This comment is a little confusing because this code doesn't reference any "security name." I think adding the security group IDs to the launch template is enough context as to what is happening here.